proto/grpc/device_testing.proto

```diff
@@ -0,0 +1,22 @@
+/*
+ * gRPC service to receive device testing results
+ */
+
+syntax = "proto3";
+
+import "forch/proto/shared_constants.proto";
+
+/*
+ * gRPC service to receive testing results
+ */
+service DeviceTesting {
+    rpc ReportTestingResult (DeviceTestingResult) returns (Empty) {}
+}
+
+/*
+ * Device testing result
+ */
+message DeviceTestingResult {
```

Comment:
I think this should be DeviceTestingState, not result. e.g., what happens if there's a set of predefined states in a file?  Exact same structure, just not a result anymore.

Reply:
Right, changed to DeviceTestingState.

Reply:
Maybe DeviceTestingState is not a good name... Will udpate..